src/pages/sns.tsx

```diff
@@ -0,0 +1,480 @@
+                promoting or participating in governance.{" "}
+                <span className="italic">No fiat required.</span>
+              </motion.p>
+              <MotionLink
+                className="link-external"
+                href="https://internetcomputer.org/docs/current/tokenomics/sns/sns-intro-tokens"
+                variants={transitions.item}
+              >
+                Learn more
+              </MotionLink>
+            </Card>
+            <Card title="Scalable" icon={<IconScalable aria-hidden="true" />}>
+              <motion.p
+                className="mb-6 tw-paragraph"
+                variants={transitions.item}
+              >
+                Infnitely scalable. The Internet Computer is the only blockchain
+                that can scale without bounds - it can truly serve the world.
+                Infinite scaling is possible through advanced cryptography which
+                joins many subnets together into a singular, secure blockchain.
+                This means there’s no limit to the volume of users a dapp can
+                service, and smart contracts are capable of much more than
+                simple token ledgers or financial contracts. They can serve full
+                stack web applications or dapps at web speed.{" "}
+                <span className="italic">
+                  The Internet Computer is a world computer.
+                </span>
+              </motion.p>
+              <MotionLink
+                className="link-external"
+                href="https://internetcomputer.org/features/limitless-scaling"
+                variants={transitions.item}
+              >
+                Learn more
+              </MotionLink>
+            </Card>
+          </div>
+        </section>
+        <section className="text-white relative pt-52 pb-20 md:pb-40 md:pt-80 container-12">
+          <AnimateSpawn
+            el={motion.img}
+            variants={transitions.fadeIn}
+            src={BlobGradient}
+            alt=""
+            className="max-w-none w-[1200px] md:w-[1600px] absolute top-[-200px] md:top-[-200px] left-1/2 -translate-x-1/2 z-[-1]"
+          />
+          <AnimateSpawn
+            className="mx-auto text-center sm:w-6/12 mb-16 md:mb-8"
+            variants={transitions.container}
+          >
+            <motion.h2
+              className="tw-heading-4 md:tw-heading-60 mb-3 md:mb-8"
+              variants={transitions.item}
+            >
+              Turn your App into a dApp with an SNS
+            </motion.h2>
+            <motion.p
+              className="tw-lead-sm md:tw-lead mb-8"
+              variants={transitions.item}
+            ></motion.p>
+            <MotionLink
+              variants={transitions.item}
+              className="button-outline-white"
+              href="/docs/current/developer-docs/integrations/sns/sns-intro#how-to-get-and-maintain-a-dao---different-options"
```

Comment:
This page is not yet public.
Recommend to link to https://internetcomputer.org/docs/current/tokenomics/sns/sns-intro-tokens in the meantime